crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -212,8 +210,6 @@ pub enum SuiSubcommand {
     ViewCheckpoint,
     #[clap(name = "run-graphql")]
     RunGraphql(RunGraphqlCommand),
-    #[clap(name = "view-graphql-variables")]
-    ViewGraphqlVariables,
```

Comment:
might still be nice to keep this, or expose the usable variables some other way

Reply:
I agree, but looking at what constitutes the "usable variables", it could be different from the set of GraphQL variable (at the moment, the main difference is the opaque cursor support for interpolation) so it would be better to implement that from a single source of truth, when we need it.